Sources/Layouts/TiltedTabTiltedCollectionViewLayout.swift

```diff
@@ -12,7 +12,8 @@ class TiltedTabTiltedCollectionViewLayout: TiltedTabCollectionViewLayout {
     private var layoutAttributes: [IndexPath: UICollectionViewLayoutAttributes] = [:]
     private var contentHeight: CGFloat = 0
     
-    private let standardAngleOfRotation: CGFloat = -50
+    private let minimumAngle: CGFloat = -30
+    private let maximumAngle: CGFloat = -80
```

Comment:
Any reason you went +30 above the original value for the maximum, but -20 for the minimum?

Reply:
This was mostly just me trying a bunch of different values to see what looked closest to Safari. Feel free to change them!